measure_extinction/plotting/plot_spec.py

```diff
@@ -389,20 +426,24 @@ def plot_spectrum(
         type=str,
         default=[],
     )
+    parser.add_argument(
+        "--log", help="plot wavelengths on a log-scale", action="store_true"
+    )
     parser.add_argument("--pdf", help="save figure as a pdf file", action="store_true")
     args = parser.parse_args()
 
     if args.onefig:  # plot all spectra in the same figure
         plot_multi_spectra(
-            args.starlist,
-            args.path,
-            args.mlam4,
-            args.HI_lines,
-            args.range,
-            args.norm_range,
-            args.spread,
-            args.exclude,
-            args.pdf,
+            starlist=args.starlist,
+            path=args.path,
```

Comment:
These are not keyword arguments, unlike the rest.  So, they shouldn't be in keyword format.